pom.xml

```diff
@@ -262,7 +262,7 @@
     <!-- Important: Selenium version 2.46.0 is the last one supported on Java 6! -->
     <version.org.jboss.arquillian.selenium>2.46.0</version.org.jboss.arquillian.selenium>
     <version.org.jboss.as.arquillian>7.2.0.Final</version.org.jboss.as.arquillian>
-    <version.org.jboss.as>7.5.4.Final-redhat-4</version.org.jboss.as>
+    <version.org.jboss.as>7.5.6.Final-redhat-2</version.org.jboss.as>
```

Comment:
I see. It seems that jboss-as-arquillian isn't in the eap support list. See: http://download.eng.pnq.redhat.com/released/JBEAP-6/6.4.6/jboss-eap-6.4.6-supported-GAVs.txt

Thanks for checking.